src/gluonts/mx/distribution/transformed_distribution.py

```diff
@@ -210,9 +210,9 @@ def __init__(
     @property
     def mean(self) -> Tensor:
         return (
-            self.base_distribution.mean
+            self.base_distribution.mean * self.scale
```

Comment:
This does not account for the case `self.scale = None`. You could use `scale` defined as

```python
scale = self.scale if self.scale is not None else 1
```